internal/controller/ai_gateway_route.go

```diff
@@ -79,15 +79,44 @@ func (c *aiGatewayRouteController) Reconcile(ctx context.Context, req reconcile.
 
 	if err := c.ensuresExtProcConfigMapExists(ctx, &aiGatewayRoute); err != nil {
 		logger.Error(err, "Failed to reconcile extProc config map")
+		aiGatewayRoute.Status.Conditions = append(aiGatewayRoute.Status.Conditions, metav1.Condition{
+			Type:    "Reconciled",
```

Comment:
Can we use constants for those:

```go
const (
    ConditionReconciled   = "Reconciled"
    ReasonConfigMapError  = "ConfigMapError"
)
```

Same for all the repetitions below?

Reply:
Sure.